content/post/28-js-ipfs-0.25.md

```diff
@@ -0,0 +1,47 @@
+---
+date: 2017-07-12
+url: 28-js-ipfs-0-25
+title: js-ipfs 0.25.0 released
+author: David Dias
+---
+
+Today, we've released js-ipfs 0.25.0. This release is mostly maintenance, bug fixing and bringing some of the internal components up to their latest versions.
+
+<blockquote class="twitter-tweet" data-conversation="none" data-lang="en"><p lang="en" dir="ltr">Good morning world 🌞<br><br>A fresh new js-ipfs is waiting for you:<br>+ ipfs@0.25.0<br>[08:52:21] Published to npm.<br><br>I bid you all a good day!</p>&mdash; David Dias (@daviddias) <a href="https://twitter.com/daviddias/status/885044398328819712">July 12, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+# 🔦 Highlights
+
+### 🏮 ipfs-repo is now 🐲 free
+
+The IPFS Repo Class went through [some careful refactoring process](https://github.com/ipfs/js-ipfs-repo/pull/140). Now it is easier to understand its internals and to support different backend storage adapters.
+
+To create a js-ipfs node with your own custom repo you just have to:
```

Comment:
"your own custom repo, you can now:" (if this is a new improvement/simplification?)